### PR TITLE
Refactor/customers details page

### PR DESCRIPTION
**Pull request by Njord201**
I've had refactor the customers details page.

Expected :
image

Result:
image

### CODE REVIEW

File: angular-frontend/src/app/pages/client-profile/client-profile.component.scss [hunk 1]
```diff
@@ -23,3 +23,8 @@ table {
     border-radius: 10px;
     overflow: hidden;
 }
+
+.image-card-payment {
+  width: auto;
+  height: 30px;
+}
```
Comment (AxelF44) [inline angular-frontend/src/app/pages/client-profile/client-profile.component.scss:30]:
```suggestion
}

```

**Review by AxelF44 — CHANGES_REQUESTED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by AxelF44 — DISMISSED** [review]

File: angular-frontend/src/app/pages/client-profile/client-profile.component.html [hunk 2]
```diff
@@ -1,92 +1,121 @@
 <div class="column is-11">
-    <p class="title is-1">{{ "navCustomers" | transloco }}</p>
+    <h2 class="m-4 fw-bold">Customer Details</h2>
     <hr>
```
Comment (mathieurobert1) [inline angular-frontend/src/app/pages/client-profile/client-profile.component.html:3]: Remove please

**Review by mathieurobert1 — CHANGES_REQUESTED** [review]

**Comment by mathieurobert1** [comment]
Also fix Fix responsive.

**Review by AxelF44 — APPROVED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by AxelF44 — APPROVED** [review]

**Review by theophile-jr — APPROVED** [review]
Brilliant !

**Pull request merged**